src/data/feed/quandl.js

```diff
@@ -0,0 +1,146 @@
+import d3 from 'd3';
+
+//  https://www.quandl.com/docs/api#datasets
+export default function() {
+
+    var _columnNameMap = function(colName) {
+        colName = colName.replace(/[\s()-.]/g, '');
```

Comment:
Sure, the default mapping would still sort Date to date and all fields for an OHLC data set so consumer would probably be doing post processing anyway.